src/Symfony/Component/DependencyInjection/ParameterBag/ContainerBagInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\ParameterBag;
+
+use Psr\Container\ContainerInterface;
+
+/**
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+interface ContainerBagInterface extends ContainerInterface
+{
+    /**
+     * Gets the service container parameters.
+     *
+     * @return array An array of parameters
+     */
+    public function all();
```

Comment:
return type declaration?

Reply:
Not possible: would make the interface incompatible with ParameterBagInterface.

Reply:
I'm too late but, is this true?
https://3v4l.org/NPigR

Reply:
Looks like it isn't.